autogen/math_utils.py

```diff
@@ -25,13 +25,13 @@ def solve_problem(problem: str, **config) -> str:
 
 
 def remove_boxed(string: str) -> Optional[str]:
-    """Source: https://github.com/hendrycks/math
-    Extract the text within a \\boxed{...} environment.
+    r"""Source: https://github.com/hendrycks/math
+    Extract the text within a \\boxed\{...\} environment.
     Example:
 
-    > remove_boxed("\\boxed{\\frac{2}{3}}")
-
-    \\frac{2}{3}
+    > remove_boxed("\\boxed\{\\frac\{2\}\{3\}\}")
+    Output:
```

Comment:
Why add the string "Output"? @kevin666aa FYI.

Reply:
I added it to improve readability as it is the output of the function called, but that's subjective. I can remove it if you find it confusing.

Reply:
@qingyun-wu how to proceed with this? I am fine with both keeping and removing the string "Output", but would like to have this merged soon so I can start working on docs versioning.